lib/imagefs.coffee

```diff
@@ -280,3 +327,13 @@ exports.listDirectory = (definition) ->
 			listDirectory(disk, definition.partition, definition.path)
 	else if definition.image instanceof filedisk.Disk
 		listDirectory(definition.image, definition.partition, definition.path)
+
+###*
+# @summary Closes the allocated resources. Call this when you're done using imagefs if you expect the program to end.
+# @function
+# @public
+#
+# @returns {Promise}
+###
+exports.close = ->
+	ext2fs.close()
```

Comment:
Is there a need to tear down the FAT fs as well (if not, a comment would be in order)?
Also, keeping the `ext2fs` a module global in here will break all over the place if this is used on two images at the same time, due its asynchronous nature.